src/testSetup/groovy/com/bmuschko/gradle/docker/TestConfiguration.groovy

```diff
@@ -1,14 +1,14 @@
 package com.bmuschko.gradle.docker
 
 final class TestConfiguration {
-    private static final String DOCKER_HOST_SYS_PROP = 'dockerHost'
+    private static final String DOCKER_HOST_SYS_PROP = 'dockerServerUrl'
```

Comment:
This was not actually being used :) We would check for the existence of `dockerHost` and since it didn't exist we fall back to `http://localhost:2375`. This failed when trying to use unix domain sockets as the host for functional tests kept getting set to `http://localhost:2375`.